homework/password-check/validation.cpp

```diff
@@ -1,2 +1,62 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+#include <string>
+#include <iostream>
+#include <cstdlib>
+#include <algorithm>
+#include <cctype>
+
+std::string getErrorMessage(const ErrorCode err)
+{
+	switch (err) 
+	{
+		case ErrorCode::Ok: 
+			return "Ok";
+		case ErrorCode::PasswordNeedsAtLeastNineCharacters: 
+			return "Password needs to have at least nine characters";
+		case ErrorCode::PasswordNeedsAtLeastOneNumber: 
+			return "Password needs to have at least one number";
+		case ErrorCode::PasswordNeedsAtLeastOneSpecialCharacter: 
+			return "Password needs to have at least one special character";
+		case ErrorCode::PasswordNeedsAtLeastOneUppercaseLetter: 
+			return "Password needs to have at least one uppercase letter";
+		case ErrorCode::PasswordsDoNotMatch: 
+			return "Passwords do not match";
```

Comment:
You can make this string as constexpr. It will be easier write tests. Because you will have one variable and don't need to copy multiple times the same strings, which could change later and then you need copy it again.